main.go

```diff
@@ -24,7 +24,7 @@ import (
 	"path/filepath"
 	"strings"
 	"syscall"
-	template_text "text/template"
+	texttemplate "text/template"
```

Comment:
Don't feel strongly here.

So template first, but I'll make it consistent with the other case, and keep it abbreviated.